src/LibraryManager.Vsix/UI/Controls/PackageContentsTreeView.xaml

```diff
@@ -10,19 +10,38 @@
              d:DesignHeight="300" d:DesignWidth="300" d:DataContext="{d:DesignInstance IsDesignTimeCreatable=False, Type=models:InstallDialogViewModel}">
     <UserControl.Resources>
         <converters:BoldingConverter x:Key="BoldingConverter" />
+        <converters:CheckBoxAutomationNameConverter x:Key="CheckBoxAutomationNameConverter" />
         <converters:PackageItemIconConverter x:Key="IconConverter" />
         <converters:VisibleIfNonNullConverter x:Key="VisibleIfNonNullConverter" />
     </UserControl.Resources>
-    <TreeView AutomationProperties.Name="{x:Static resources:Text.Files}"
-              ItemsSource="{Binding Path=DisplayRoots, Mode=OneWay}"
+    <TreeView ItemsSource="{Binding Path=DisplayRoots, Mode=OneWay}"
               VirtualizingPanel.IsVirtualizing="True"
-              KeyboardNavigation.TabNavigation="Continue">
+              KeyboardNavigation.TabNavigation="Continue"
+              PreviewKeyUp="OnPreviewKeyUp">
         <TreeView.ItemContainerStyle>
             <Style TargetType="{x:Type TreeViewItem}" d:DataContext="{d:DesignInstance Type=TreeViewItem, IsDesignTimeCreatable=False}">
                 <Setter Property="IsExpanded" Value="{Binding Path=IsExpanded, Mode=TwoWay}" />
-                <Setter Property="Focusable" Value="False" />
-                <Setter Property="KeyboardNavigation.TabNavigation" Value="Continue"/>
-                <Setter Property="AutomationProperties.Name" Value="{Binding Name}"/>
+                <Setter Property="KeyboardNavigation.TabNavigation" Value="Once"/>
+                <Setter Property="AutomationProperties.Name">
+                    <Setter.Value>
+                        <MultiBinding Converter="{StaticResource CheckBoxAutomationNameConverter}">
```

Comment:
It seems weird that the state of the checkbox is bound into the name.  Is this a hack to workaround the control not being focusable?

Reply:
The issue here is the treeview item has a checkbox and label bound to it. We can't give focus to checkbox as it would take additional key press event and every keyUp, keyDown would take two key press instead of one. Let me know if you have any suggestions.